nixos/modules/services/networking/mptcp/mptcp_up_raw

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+# A script for setting up routing tables for MPTCP
+
+# can check with (ins)[root@client:~]# journalctl -b -u NetworkManager-dispatcher.service
+# Copy this script into /etc/network/if-up.d/
+
+set -ex
+
+STATUS=$2
+
+# TODO remove
+echo $PATH
+env > /tmp/if_up_env
```

Comment:
Insecure use of temporary files. `/tmp/if_up_env` could be an attacker controlled symlink.

Reply:
He said in the comment that he is going to remove that.

Reply:
I still added the comment so it won't be forgotten.